Jenkinsfile

```diff
@@ -50,6 +50,7 @@ def prepareEnv(branch, remote, distroVersion, functionalTests, platform) {
       if (functionalTests.contains('FVT')) {
           if (platform == "Azure") {
               env.LISAV2_PARAMS = "-TestCategory 'Functional,Community,Stress,BVT' -TestArea 'KVP,SRIOV,NETWORK,STORAGE,WALA,CORE,KDUMP,LTP,STRESS,BVT,NVME'"
+              LISAV2_AZURE_REGION = "eastus2"
```

Comment:
you are changing the region for all the functional tests, please change the commit message.

Reply:
done, thanks for review